src/cmd/manage_nsfs.js

```diff
@@ -690,6 +674,56 @@ async function get_config_data(config_file_path, show_secrets = false) {
     return config_data;
 }
 
+/**
+ * get_options_from_file will read a JSON file that include key-value of the options 
+ * (instead of flags) and return its content
+ * @param {string} type
+ * @param {string} action
+ * @param {string} file_path
+ */
+async function get_options_from_file(type, action, file_path) {
+    const fs_context = native_fs_utils.get_process_fs_context();
+    try {
+        const raw_data = (await nb_native().fs.readFile(fs_context, file_path)).data;
+        const input_options_with_data = JSON.parse(raw_data.toString());
+        const input_options = Object.keys(input_options_with_data);
+        if (input_options.includes(FROM_FILE)) {
+            const details = `${FROM_FILE} should not be passed inside json options`;
+            throw_cli_error(ManageCLIError.InvalidArgument, details);
+        }
+        validate_no_extra_options(type, action, input_options);
+        validate_options_type_by_value(input_options_with_data);
+        return input_options_with_data;
+    } catch (err) {
+        if (err.code === 'ENOENT') throw_cli_error(ManageCLIError.InvalidFilePath, file_path);
+        if (err instanceof SyntaxError) throw_cli_error(ManageCLIError.InvalidJSONFile, file_path);
+        throw err;
+    }
+}
+
+/**
+ * get_access_keys will return the access_keys and new_access_key according to the user input
+ * and action
+ * @param {string} action
+ * @param {object} user_input
+ */
+function get_access_keys(action, user_input) {
+    let access_keys = [{
+        access_key: undefined,
+        secret_key: undefined,
+    }];
+    let new_access_key;
+    if (action !== ACTIONS.LIST && action !== ACTIONS.STATUS) _validate_access_keys(user_input.access_key, user_input.secret_key);
+    if (action === ACTIONS.ADD || action === ACTIONS.STATUS) {
```

Comment:
You didn't write this function but this whole algorithm of this function is a bit weird, let's try focus on this function on a different PR if needed, I'm bothered about generation/regeneration of credentials flows like this that are integrated with simple fetching from user flows

Reply:
Do you want to open an issue?
It will allow us to focus on what might bother you.

Reply:
sure, I wrote it in the tracking sheet

Reply:
Resolving, it is a GAP in the tracking sheet.